### PR TITLE
Fix for K8SPXC-706: moving from self-signed to CA-signed certificates with cert-manager

### PR DESCRIPTION
**Pull request by arronax**
[K8SPXC-706](https://jira.percona.com/browse/K8SPXC-706) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

As explained in https://jira.percona.com/browse/K8SPXC-706, current
default PXC operator deployment will have issues upon certificate
renewal. To move from self-signed certificates to CA-signed
certificates, a non-self-signed Issuer or ClusterIssuer is required.

This commit changes operator's logic of working with Cert Manager. Now
operator will create a self-signed issuer, then a self-signed cert, and
then use that cert to create a CA issuer. PXC certificates are then
generated by a CA issuer, and their renewal works fine.

It is impossible to create a cluster issuer, because that requires
placing both a self-signed issuer and a CA certificate into cert-manager
namespace, which is not allowed for the current operator's user.

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| recreate | passed |
| big-data | passed |
| scheduled-backup | passed |
| storage | passed |
| upgrade-haproxy | passed |
| scaling | passed |
| haproxy | passed |
| init-deploy | failed |
| self-healing | passed |
| scaling-proxysql | passed |
| upgrade-proxysql | passed |
| restore-to-encrypted-cluster | passed |
| security-context | passed |
| self-healing-advanced | passed |
| smart-update | passed |
| demand-backup | passed |
| limits | passed |
| operator-self-healing | passed |
| monitoring-2-0 | failed |
| affinity | passed |
| pitr | passed |
| one-pod | passed |
| upgrade-consistency | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| demand-backup-encrypted-with-tls | passed |
| tls-issue-self | passed |
| tls-issue-cert-manager | failed |
| tls-issue-cert-manager-ref | failed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/886/commits/c3faf957f4fc2d0d51161ec874d70be158d9abe8
image: `perconalab/percona-xtradb-cluster-operator:PR-886-c3faf957`

File: pkg/controller/pxc/tls.go [hunk 1]
```diff
@@ -180,7 +210,26 @@ func (r *ReconcilePerconaXtraDBCluster) createIssuer(ownRef []metav1.OwnerRefere
...
+func (r *ReconcilePerconaXtraDBCluster) createIssuer(ownRef []metav1.OwnerReference, namespace, issuer string, caCertSecret string) error {
+	err := r.client.Create(context.TODO(), &cm.Issuer{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:            issuer,
+			Namespace:       namespace,
+			OwnerReferences: ownRef,
+		},
+		Spec: cm.IssuerSpec{
+			IssuerConfig: cm.IssuerConfig{
+				CA: &cm.CAIssuer{SecretName: caCertSecret},
```
Comment (delgod) [inline pkg/controller/pxc/tls.go:227]: Do we really need two functions with only differences in one line, which can be passed as an argument?

File: pkg/controller/pxc/tls.go [hunk 2]
```diff
@@ -57,14 +57,46 @@ func (r *ReconcilePerconaXtraDBCluster) createSSLByCertManager(cr *api.PerconaXt
 	ownerReferences := []metav1.OwnerReference{owner}
 
 	issuerName := cr.Name + "-pxc-ca"
+	selfSignedIssuerName := cr.Name + "-pxc-selfsigned"
```
Comment (delgod) [inline pkg/controller/pxc/tls.go:60]: @hors we need to come up with better names for two issuers and two certificates.
also, we must check how these changes affect clusters during update from 1.8.0 to 1.9.0

**Review by delgod — CHANGES_REQUESTED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scheduled-backup | passed |
| haproxy | passed |
| big-data | passed |
| recreate | passed |
| storage | passed |
| upgrade-haproxy | passed |
| scaling | passed |
| init-deploy | failed |
| self-healing | passed |
| scaling-proxysql | passed |
| restore-to-encrypted-cluster | passed |
| security-context | passed |
| upgrade-proxysql | passed |
| self-healing-advanced | passed |
| demand-backup | passed |
| smart-update | passed |
| limits | passed |
| operator-self-healing | passed |
| monitoring-2-0 | failed |
| pitr | passed |
| affinity | passed |
| one-pod | passed |
| upgrade-consistency | passed |
| auto-tuning | passed |
| demand-backup-encrypted-with-tls | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| tls-issue-self | passed |
| tls-issue-cert-manager | failed |
| tls-issue-cert-manager-ref | failed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/886/commits/7e8207d7af698afcd342f1eb4389cfe82a26aa8b
image: `perconalab/percona-xtradb-cluster-operator:PR-886-7e8207d7`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| big-data | passed |
| scheduled-backup | passed |
| recreate | passed |
| haproxy | passed |
| scaling | passed |
| storage | passed |
| init-deploy | failed |
| scaling-proxysql | passed |
| self-healing | passed |
| restore-to-encrypted-cluster | passed |
| security-context | passed |
| self-healing-advanced | passed |
| demand-backup | passed |
| limits | passed |
| operator-self-healing | passed |
| monitoring-2-0 | failed |
| pitr | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| demand-backup-encrypted-with-tls | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| tls-issue-self | passed |
| tls-issue-cert-manager | failed |
| tls-issue-cert-manager-ref | failed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/886/commits/be3d4feb30ec2ebf2924ef0641f3fb95ccf24f18
image: `perconalab/percona-xtradb-cluster-operator:PR-886-be3d4feb`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scheduled-backup | passed |
| upgrade-haproxy | passed |
| storage | passed |
| self-healing | passed |
| self-healing-advanced | passed |
| operator-self-healing | passed |
| upgrade-proxysql | passed |
| smart-update | passed |
| upgrade-consistency | passed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/886/commits/be3d4feb30ec2ebf2924ef0641f3fb95ccf24f18
image: `perconalab/percona-xtradb-cluster-operator:PR-886-be3d4feb`

File: pkg/controller/pxc/tls.go [hunk 3]
```diff
@@ -56,15 +56,49 @@ func (r *ReconcilePerconaXtraDBCluster) createSSLByCertManager(cr *api.PerconaXt
...
+				SecretName: cr.Name + "-ca-cert",
+				CommonName: cr.Name + "-ca",
+				IsCA:       true,
+				IssuerRef: cmmeta.ObjectReference{
+					Name:  caIssuerName,
+					Kind:  issuerKind,
+					Group: issuerGroup,
+				},
+				Duration:    &metav1.Duration{Duration: 87600 * time.Hour},
+				RenewBefore: &metav1.Duration{Duration: 730 * time.Hour},
```
Comment (delgod) [inline pkg/controller/pxc/tls.go:88]: I think renewal every 30 days can lead to issues every 30 days.
```suggestion
				RenewBefore: &metav1.Duration{Duration: 78840 * time.Hour},
```

**Review by delgod — CHANGES_REQUESTED** [review]

File: pkg/controller/pxc/tls.go (hunk 3 above)
Comment (arronax) [reply pkg/controller/pxc/tls.go:88]: @delgod that's 30 days before the expiration (in 10 years) https://cert-manager.io/docs/usage/certificate/#renewal

Comment (hors) [reply pkg/controller/pxc/tls.go:88]: In doc we need to add information that in 10 years client will have the issue with CA certificate and the maintenance should be scheduled to renew it :)

**Review by qJkee — APPROVED** [review]

**Review by hors — CHANGES_REQUESTED** [review]
Thanks for PR but please fix the tests too. If you need any help just ping me.

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scheduled-backup | passed |
| big-data | passed |
| upgrade-haproxy | passed |
| recreate | passed |
| storage | passed |
| restore-to-encrypted-cluster | passed |
| self-healing | passed |
| demand-backup | passed |
| self-healing-advanced | passed |
| pitr | passed |
| operator-self-healing | passed |
| demand-backup-encrypted-with-tls | passed |
| haproxy | passed |
| init-deploy | failed |
| scaling | passed |
| scaling-proxysql | passed |
| security-context | passed |
| upgrade-proxysql | passed |
| smart-update | passed |
| upgrade-consistency | passed |
| limits | passed |
| monitoring-2-0 | failed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| tls-issue-self | passed |
| tls-issue-cert-manager | failed |
| tls-issue-cert-manager-ref | failed |
| validation-hook | passed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/886/commits/be3d4feb30ec2ebf2924ef0641f3fb95ccf24f18
image: `perconalab/percona-xtradb-cluster-operator:PR-886-be3d4feb`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| haproxy | passed |
| upgrade-haproxy | passed |
| recreate | passed |
| scaling | passed |
| big-data | passed |
| scheduled-backup | passed |
| init-deploy | failed |
| self-healing | passed |
| scaling-proxysql | passed |
| restore-to-encrypted-cluster | passed |
| security-context | passed |
| upgrade-proxysql | passed |
| self-healing-advanced | passed |
| demand-backup | passed |
| smart-update | passed |
| limits | passed |
| monitoring-2-0 | failed |
| pitr | passed |
| affinity | passed |
| operator-self-healing | passed |
| one-pod | passed |
| auto-tuning | passed |
| demand-backup-encrypted-with-tls | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| tls-issue-self | passed |
| tls-issue-cert-manager | failed |
| upgrade-consistency | failed |
| tls-issue-cert-manager-ref | failed |
| validation-hook | passed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/886/commits/f09e974b358f6616f14f825925c861a2dcb23e01
image: `perconalab/percona-xtradb-cluster-operator:PR-886-f09e974b`

File: pkg/controller/pxc/tls.go (hunk 3 above)
Comment (arronax) [reply pkg/controller/pxc/tls.go:88]: @hors I believe we should just explain in the docs why exactly CA renewal leads to the issues. But of course, the default 10 year mark should be mentioned, too :)

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| upgrade-haproxy | passed |
| scaling | passed |
| big-data | passed |
| storage | passed |
| scaling-proxysql | passed |
| self-healing | passed |
| security-context | passed |
| self-healing-advanced | passed |
| operator-self-healing | passed |
| haproxy | passed |
| init-deploy | failed |
| scheduled-backup | passed |
| recreate | passed |
| restore-to-encrypted-cluster | passed |
| demand-backup | passed |
| pitr | passed |
| demand-backup-encrypted-with-tls | passed |
| upgrade-proxysql | passed |
| smart-update | passed |
| upgrade-consistency | passed |
| limits | passed |
| monitoring-2-0 | failed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| tls-issue-self | passed |
| tls-issue-cert-manager | failed |
| tls-issue-cert-manager-ref | failed |
| validation-hook | passed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/886/commits/f09e974b358f6616f14f825925c861a2dcb23e01
image: `perconalab/percona-xtradb-cluster-operator:PR-886-f09e974b`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| haproxy | passed |
| init-deploy | failed |
| scaling | passed |
| scaling-proxysql | passed |
| security-context | passed |
| big-data | passed |
| scheduled-backup | passed |
| recreate | passed |
| upgrade-haproxy | passed |
| restore-to-encrypted-cluster | passed |
| demand-backup | passed |
| pitr | passed |
| storage | passed |
| demand-backup-encrypted-with-tls | passed |
| self-healing | passed |
| self-healing-advanced | passed |
| operator-self-healing | passed |
| upgrade-proxysql | passed |
| smart-update | passed |
| upgrade-consistency | passed |
| limits | passed |
| monitoring-2-0 | failed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| tls-issue-self | passed |
| tls-issue-cert-manager | failed |
| tls-issue-cert-manager-ref | failed |
| validation-hook | passed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/886/commits/f09e974b358f6616f14f825925c861a2dcb23e01
image: `perconalab/percona-xtradb-cluster-operator:PR-886-f09e974b`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/886/commits/f09e974b358f6616f14f825925c861a2dcb23e01
image: `perconalab/percona-xtradb-cluster-operator:PR-886-f09e974b`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scaling | passed |
| upgrade-haproxy | passed |
| storage | passed |
| haproxy | passed |
| scheduled-backup | passed |
| big-data | passed |
| recreate | passed |
| init-deploy | failed |
| scaling-proxysql | passed |
| self-healing | passed |
| restore-to-encrypted-cluster | passed |
| security-context | passed |
| upgrade-proxysql | passed |
| self-healing-advanced | passed |
| demand-backup | passed |
| smart-update | passed |
| operator-self-healing | passed |
| limits | passed |
| monitoring-2-0 | failed |
| pitr | passed |
| affinity | passed |
| one-pod | passed |
| upgrade-consistency | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | failed |
| demand-backup-encrypted-with-tls | failed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/886/commits/d0529d49f2ff8d6cc08b23baf68fa8367e1efb75
image: `perconalab/percona-xtradb-cluster-operator:PR-886-d0529d49`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scheduled-backup | passed |
| storage | passed |
| scaling | passed |
| upgrade-haproxy | passed |
| recreate | passed |
| haproxy | passed |
| big-data | passed |
| init-deploy | failed |
| scaling-proxysql | passed |
| self-healing | passed |
| restore-to-encrypted-cluster | passed |
| security-context | passed |
| upgrade-proxysql | passed |
| self-healing-advanced | passed |
| demand-backup | passed |
| smart-update | passed |
| limits | passed |
| monitoring-2-0 | failed |
| operator-self-healing | passed |
| pitr | passed |
| affinity | passed |
| one-pod | passed |
| upgrade-consistency | passed |
| auto-tuning | passed |
| demand-backup-encrypted-with-tls | passed |
| proxysql-sidecar-res-limits | passed |
| users | failed |
| tls-issue-self | failed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/886/commits/9a52ccabd025aa4b5cafff756f23853a92f1c8c0
image: `perconalab/percona-xtradb-cluster-operator:PR-886-9a52ccab`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| big-data | passed |
| scheduled-backup | failed |
| scaling | passed |
| haproxy | passed |
| storage | passed |
| upgrade-haproxy | passed |
| recreate | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| init-deploy | failed |
| security-context | passed |
| limits | passed |
| restore-to-encrypted-cluster | passed |
| self-healing-advanced | passed |
| upgrade-proxysql | passed |
| monitoring-2-0 | failed |
| demand-backup | passed |
| smart-update | failed |
| operator-self-healing | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| tls-issue-self | passed |
| upgrade-consistency | failed |
| tls-issue-cert-manager | failed |
| pitr | passed |
| tls-issue-cert-manager-ref | failed |
| demand-backup-encrypted-with-tls | passed |
| validation-hook | passed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/886/commits/f687aeccda8278dd96cc72987f9a7df728734d1e
image: `perconalab/percona-xtradb-cluster-operator:PR-886-f687aecc`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| haproxy | passed |
| scaling | passed |
| recreate | passed |
| storage | passed |
| big-data | passed |
| scheduled-backup | passed |
| upgrade-haproxy | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| init-deploy | failed |
| restore-to-encrypted-cluster | passed |
| security-context | passed |
| upgrade-proxysql | passed |
| self-healing-advanced | passed |
| smart-update | passed |
| operator-self-healing | passed |
| limits | passed |
| monitoring-2-0 | failed |
| demand-backup | passed |
| affinity | passed |
| one-pod | passed |
| upgrade-consistency | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| pitr | passed |
| tls-issue-self | passed |
| tls-issue-cert-manager | failed |
| demand-backup-encrypted-with-tls | failed |
| tls-issue-cert-manager-ref | failed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/886/commits/0d73acc4a79fa61cb52cb877e68c17b97f017ac9
image: `perconalab/percona-xtradb-cluster-operator:PR-886-0d73acc4`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scaling | passed |
| storage | passed |
| haproxy | passed |
| upgrade-haproxy | passed |
| recreate | passed |
| scheduled-backup | passed |
| big-data | passed |
| init-deploy | failed |
| self-healing | failed |
| scaling-proxysql | passed |
| restore-to-encrypted-cluster | passed |
| security-context | passed |
| upgrade-proxysql | passed |
| limits | passed |
| demand-backup | passed |
| monitoring-2-0 | failed |
| smart-update | passed |
| self-healing-advanced | passed |
| affinity | passed |
| operator-self-healing | passed |
| one-pod | passed |
| auto-tuning | passed |
| pitr | passed |
| proxysql-sidecar-res-limits | passed |
| users | failed |
| demand-backup-encrypted-with-tls | failed |
| upgrade-consistency | passed |
| tls-issue-self | failed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/886/commits/68e34cf20179963406f7e5e0803a0b67b9c1f69a
image: `perconalab/percona-xtradb-cluster-operator:PR-886-68e34cf2`

**Review by delgod — APPROVED** [review]

**Review by qJkee — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| haproxy | passed |
| init-deploy | failed |
| upgrade-haproxy | passed |
| scheduled-backup | passed |
| big-data | passed |
| scaling | passed |
| recreate | passed |
| storage | passed |
| scaling-proxysql | passed |
| restore-to-encrypted-cluster | passed |
| self-healing | failed |
| security-context | passed |
| demand-backup | passed |
| pitr | passed |
| demand-backup-encrypted-with-tls | passed |
| upgrade-proxysql | passed |
| smart-update | passed |
| upgrade-consistency | passed |
| limits | passed |
| monitoring-2-0 | failed |
| self-healing-advanced | passed |
| operator-self-healing | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | failed |
| tls-issue-self | failed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/886/commits/68e34cf20179963406f7e5e0803a0b67b9c1f69a
image: `perconalab/percona-xtradb-cluster-operator:PR-886-68e34cf2`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| recreate | passed |
| upgrade-haproxy | passed |
| scheduled-backup | passed |
| storage | passed |
| big-data | passed |
| haproxy | passed |
| scaling | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| init-deploy | passed |
| restore-to-encrypted-cluster | passed |
| security-context | passed |
| limits | passed |
| upgrade-proxysql | passed |
| monitoring-2-0 | passed |
| self-healing-chaos | passed |
| affinity | passed |
| smart-update | failed |
| one-pod | passed |
| self-healing-advanced | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| self-healing-advanced-chaos | passed |
| demand-backup | passed |
| users | passed |
| operator-self-healing | passed |
| upgrade-consistency | passed |
| operator-self-healing-chaos | passed |
| pitr | passed |
| tls-issue-self | passed |
| tls-issue-cert-manager | failed |
| tls-issue-cert-manager-ref | passed |
| validation-hook | passed |
| demand-backup-encrypted-with-tls | failed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/886/commits/5931cec66d314ac52362a7b1c72f9dace640ff23
image: `perconalab/percona-xtradb-cluster-operator:PR-886-5931cec6`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| big-data | passed |
| upgrade-haproxy | passed |
| recreate | passed |
| scaling | passed |
| scheduled-backup | passed |
| haproxy | passed |
| storage | passed |
| scaling-proxysql | passed |
| self-healing | passed |
| security-context | passed |
| restore-to-encrypted-cluster | passed |
| init-deploy | passed |
| self-healing-chaos | passed |
| demand-backup | passed |
| limits | passed |
| upgrade-proxysql | failed |
| self-healing-advanced | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| self-healing-advanced-chaos | passed |
| one-pod | passed |
| auto-tuning | passed |
| smart-update | failed |
| pitr | passed |
| operator-self-healing | passed |
| proxysql-sidecar-res-limits | passed |
| users | failed |
| operator-self-healing-chaos | passed |
| upgrade-consistency | passed |
| demand-backup-encrypted-with-tls | failed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/886/commits/c5e7edbcee7541162659d5f16f55bdd883727aa1
image: `perconalab/percona-xtradb-cluster-operator:PR-886-c5e7edbc`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scheduled-backup | passed |
| recreate | passed |
| restore-to-encrypted-cluster | passed |
| demand-backup | passed |
| pitr | passed |
| demand-backup-encrypted-with-tls | passed |
| big-data | passed |
| upgrade-haproxy | passed |
| haproxy | passed |
| storage | passed |
| scaling | passed |
| init-deploy | passed |
| scaling-proxysql | passed |
| self-healing | passed |
| limits | passed |
| self-healing-chaos | passed |
| security-context | passed |
| monitoring-2-0 | passed |
| self-healing-advanced | passed |
| affinity | passed |
| self-healing-advanced-chaos | passed |
| one-pod | passed |
| operator-self-healing | passed |
| auto-tuning | passed |
| operator-self-healing-chaos | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| upgrade-proxysql | failed |
| smart-update | failed |
| upgrade-consistency | passed |
| tls-issue-self | passed |
| tls-issue-cert-manager | passed |
| tls-issue-cert-manager-ref | passed |
| validation-hook | passed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/886/commits/c5e7edbcee7541162659d5f16f55bdd883727aa1
image: `perconalab/percona-xtradb-cluster-operator:PR-886-c5e7edbc`

**Review by hors — APPROVED** [review]

**Review by qJkee — APPROVED** [review]

**Pull request merged**